daemon/list.go

```diff
@@ -668,3 +685,56 @@ func populateImageFilterByParents(ancestorMap map[image.ID]bool, imageID image.I
 		ancestorMap[imageID] = true
 	}
 }
+
+// Events returns events generated by daemon
```

Comment:
I am afraid  yes. Since code here
```

if err := enc.Encode(jev); err != nil {
        return err
}
```
will encode the events into http.ResponseWriter.

Correct me if I missed something.